pkg/process/v6/transformers/nvd/transform.go

```diff
@@ -157,6 +161,24 @@ func getAffected(cfg Config, vulnerability unmarshal.NVDVulnerability) []grypeDB
 	return affs
 }
 
+func getCWEs(vulnerability unmarshal.NVDVulnerability) []grypeDB.CWEHandle {
+	var cwes []grypeDB.CWEHandle
+	for _, w := range vulnerability.Weaknesses {
+		for _, d := range w.Description {
+			if d.Value == "NVD-CWE-noinfo" || d.Value == "" {
```

Comment:
There was formerly some more complex validation here, checking against a regex like `CWE-\d+` or have the exact value `NVD-CWE-Other`. Since this is an uninterpreted string value in our data model, I don't see the need for further validation. Put another way: I don't want to change a regex if NVD comes up with another special CWE value besides `NVD-CWE-noinfo` and `NVD-CWE-Other`.

Here's a query:

```
❯ sqlite3 ./build/vulnerability.db "SELECT cwe, COUNT(*) AS occurrences FROM cwe_handles WHERE cwe NOT REGEXP '^CWE-\d+$' GROUP BY cwe ORDER BY occurrences DESC;"
NVD-CWE-Other|29673
```